Khan/perseus
# Update categorizer to have no empty "th" cells per a11y audit

## Summary:
->Not merging until Perseus has been released<-

Based on an a11y audit completed by Level Access, table headers that are empty should use "td" instead of "th" elements. This PR updates the table in categorizer to follow this a11y best practice. It also updates several related snapshots.

After:
<img width="746" alt="image" src="https://user-images.githubusercontent.com/60857422/211905709-3e3ecb03-d841-411c-b494-31694205bd11.png">

Issue: LP-13085

## Test plan:
- Confirm the first empty cell in the header row is a "td" cell instead of a "th" cell
- Confirm the appearance is the same for the before and after
- Confirm all checks pass and all tests pass
